src/utils/pause/frames/displayName.js

```diff
@@ -82,13 +82,22 @@ export function formatDisplayName(
   frame: LocalFrame,
   { shouldMapDisplayName = true, maxLength = 25 }: formatDisplayNameParams = {}
 ) {
-  let { displayName, originalDisplayName, library } = frame;
-  displayName = originalDisplayName || displayName;
+  let {
+    displayName,
+    originalDisplayName,
+    userDisplayName,
+    name,
+    library
+  } = frame;
+  displayName = originalDisplayName || userDisplayName || displayName || name;
   if (library && shouldMapDisplayName) {
     displayName = mapDisplayNames(frame, library);
   }
 
-  displayName = simplifyDisplayName(displayName);
+  displayName = displayName
+    ? simplifyDisplayName(displayName)
+    : L10N.getStr("anonymousFunction");
+
   return Number.isInteger(maxLength)
     ? endTruncateStr(displayName, maxLength)
```

Comment:
this is a tangent, but are we using maxLength anywhere now?

Reply:
Yeah, `formatCopyName` still does (it doesn't pass anything, but `maxLength` gets defaulted to 25).
Anyway, since `formatCopyName` seems only used to copy the stacktrace, I think it would be good to have the full length there too. Which means we can then remove the `maxLength` option.

I'll do that in a follow-up PR since the concern is a bit different from this one.